modules/browser-node/src/services/store.ts

```diff
@@ -310,9 +314,11 @@ export class BrowserStore implements IEngineStore, IChainServiceStore {
   }
 
   async getActiveTransactions(): Promise<StoredTransaction[]> {
-    const tx = await this.db.transactions.filter(tx => {
-      return !!tx.transactionHash && !tx.blockHash && !tx.gasUsed
-    }).toArray();
+    const tx = await this.db.transactions
+      .filter((tx) => {
+        return !!tx.transactionHash && !tx.blockHash && !tx.gasUsed;
+      })
+      .toArray();
     return tx;
   }
```

Comment:
or i guess how would this handle transactions that failed? perhaps we should dig deeper into what the txs look like when they need to be resubmitted. my guess is this would likely not get them all

Reply:
fixed (and altered) in tx_service branch https://github.com/connext/vector/pull/612

Reply:
This PR did not touch this.